manual/mingw/tools/chocolateyInstall.ps1

```diff
@@ -1,64 +1,13 @@
-﻿# Default values
-$packageName = 'mingw'
-$packageVersion = '4.8.3'
-$rev = 'rev0'
-$threads = 'posix'
-$exception = 'default' #dwarf is 32bit only, seh is 64bit only, sjlj works with 32 / 64
+﻿. (Join-Path (Split-Path -parent $MyInvocation.MyCommand.Definition) chocolateyInclude.ps1)

```

Comment:
Worked on Server 2003 and Windows 10 WIP (my test beds). I'll have to double-check `&`.